templates/macros/cycle-select.html

```diff
@@ -1,6 +1,6 @@
-{% macro cycle_select(cycles, cycle, location='query') %}
+{% macro cycle_select(cycles, cycle, location='query', id='cycle-select') %}
 {% if cycles %}
-  <select class="cycle-select select--neutral js-cycle" aria-label="Two-Year Period" name="cycle" data-cycle-location="{{ location }}">
+  <select id="{{ id }}" class="cycle-select select--neutral js-cycle" aria-label="Two-Year Period" name="cycle" data-cycle-location="{{ location }}">
```

Comment:
I added it so we could explicitly assign a `<label>`. We don't always do this, but it's nice to have there if we need it, imo.